cmd/client/testnet-observer/telegraf.conf

```diff
@@ -0,0 +1,88 @@
+# Configuration for telegraf agent
```

Comment:
is this the right repo for this to live in?

Reply:
I am unsure. Expecting suggestions :)

Reply:
since this is a specific deployment rather than the core drand system, and we eventually intend to terraform / dev-ops that structure, i'd vote for this to live in drand-infra